From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 06 Apr 2015 17:51:50 +0100
Subject: USB: Avoid ABI change in 3.2.69
Forwarded: not-needed

Hide the added bitfield usb_hcd::cant_recv_wakeups from genksyms.
---
--- a/include/linux/usb/hcd.h
+++ b/include/linux/usb/hcd.h
@@ -128,8 +128,11 @@ struct usb_hcd {
 	unsigned		wireless:1;	/* Wireless USB HCD */
 	unsigned		authorized_default:1;
 	unsigned		has_tt:1;	/* Integrated TT in root hub */
+#ifndef __GENKSYMS__
 	unsigned		cant_recv_wakeups:1;
 			/* wakeup requests from downstream aren't received */
+	/* 24 bits spare */
+#endif
 
 	int			irq;		/* irq allocated */
 	void __iomem		*regs;		/* device memory/io */
